packages/grafana-runtime/src/utils/DataSourceWithBackend.ts

```diff
@@ -101,8 +113,29 @@ export class DataSourceWithBackend<
     return getBackendSrv().post(`/api/datasources/${this.id}/resources/${path}`, { ...body });
   }
 
-  testDatasource() {
-    // TODO, this will call the backend healthcheck endpoint
-    return Promise.resolve({});
+  /**
+   * Run the datasource healthcheck
+   */
+  async callHealthCheck(): Promise<HealthCheckResult> {
+    // TODO: if the service is ERROR it returns 503... this causes a popup
```

Comment:
Since the request sends a 503 response, an error is displayed:
![image](https://user-images.githubusercontent.com/705951/76655830-776f3f80-652b-11ea-99d7-2faae89c9d11.png)

there must be some way to hide this

Reply:
@torkelo ?

Reply:
Yes ,catch error and set err.isHandled